pg/postgresql.conf

```diff
@@ -228,9 +228,9 @@ wal_level = logical			# minimal, replica, or logical
 
 # - Checkpoints -
 
-#checkpoint_timeout = 5min		# range 30s-1d
+checkpoint_timeout = 30s
 #max_wal_size = 1GB
-#min_wal_size = 80MB
+min_wal_size = 35MB
```

Comment:
**comment (non-blocking)**: You can set `fsync = off` here to [drastically speed up Postgres](https://mikecann.co.uk/posts/drastically-speed-up-your-local-postgres-tests). Since Postgres is running in a container, all the data is ephemeral anyway. So there's no point ensuring that it's flushed to disk.

Reply:
interesting. I wonder if that will help or hurt the delayed replication I was fighting earlier